### ISSUE TITLE
Add SGDRegressor

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#62]
Add SGDRegressor model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference SGDRegressor Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Linear%20Models/SGDRegressor.ipynb

Offical API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDRegressor.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an SGDRegressor as a potential best fit model.

**Comment by melan96** [comment]
I will go through it. I'll the PR as soon as possible!

**Comment by melan96** [comment]
Commit #82 with PR added for solve this issue.
